server/configuration.go

```diff
@@ -114,6 +140,7 @@ func (c *configuration) getClientConfig() clientConfig {
 		AllowEnableCalls:    c.AllowEnableCalls,
 		DefaultEnabled:      c.DefaultEnabled,
 		ICEServers:          c.ICEServers,
+		ICEServersConfigs:   c.getICEServers(),
```

Comment:
Leaving a comment here, but it applies to line 142 -- I realized that if this new version is merged, we could still have old versions of the mobile app using the new plugin. But if someone starts using the `ICEServersConfigs`, those won't be translated to old ICEServers list. (Actually, as far as I can tell, this PR removes ICEServers, so this will always be empty after this is merged, right?)

If I'm understanding this right, then we should add in a conversion to turn `ICEServersConfigs` into `ICEServers` when there isn't a user/credential field, and we need to add docs that adding a credentials server here is a breaking change for mobile apps < `1.54`.

Reply:
> Actually, as far as I can tell, this PR removes ICEServers, so this will always be empty after this is merged, right?

We are only removing the field from the settings but the values would still be returned and configurable in `config.json` if needed. To be extra thorough we could put some migration in place to convert from deprecated `ICEServers` to `ICEServersConfigs` but it seems a bit overkill considering we are in beta and those fields aren't actively used.

As for backward mobile support I think that's a good point although I'd like to highlight a couple of things: 

- STUN servers aren't necessary on the client side to make calls work. 
- TURN servers were not usable at all before this PR.

So overall there's isn't much that could break due to this change if everything is configured appropriately. Only thing that comes to mind are cases where clients rely on STUN candidates but only because the network wasn't configured appropriately (opening UDP ports). But even then, mobile code still defaulted to the hardcoded google/twilio ones so it would still work.

I think that eventually it's up to us to decide if this is a breaking enough change to justify the overhead of conversion and then deprecating the code for future removal. Thoughts?

Reply:
Thanks for clarifying! I agree, I don't think it's breaking enough to warrant adding the conversion code and the debt. Let's merge. 👍